src/Init/Data/Nat/Basic.lean

```diff
@@ -537,9 +581,14 @@ protected theorem le_iff_lt_or_eq {n m : Nat} : n ≤ m ↔ n < m ∨ n = m :=
 
 protected theorem lt_succ_iff : m < succ n ↔ m ≤ n := ⟨le_of_lt_succ, lt_succ_of_le⟩
 
+protected theorem lt_add_one_iff' : m < n + 1 ↔ m ≤ n := ⟨le_of_lt_succ, lt_succ_of_le⟩
```

Comment:
Why is there a prime here? I don't think `Nat.lt_add_one_iff` otherwise exists.

Reply:
It already exists in Mathlib with this name, I believe it should be moved to core, but I didn't know the process, so I added it with the primed name as a place holder. Knowing a little bit more now, I guess the process is to delete that theorem from [lean-pr-testing-4242](https://github.com/leanprover-community/mathlib4/compare/nightly-testing...lean-pr-testing-4242).

Reply:
That's right!